docs/tutorials/tracee-vagrant.md

```diff
@@ -0,0 +1,42 @@
+# Using the Tracee container image on MacOS with Parallels and Vagrant
```

Comment:
I'm still puzzled by why this tutorial exists, but since this pr is just moving things around let's merge it and maybe reconsider this page later

Reply:
hmmm... because the standard Tracee Docker Command does not work on M1

Using the `latest` container image:
```console
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
ERROR:
ERROR: You have to set LIBBPFGO_OSRELEASE_FILE env variable.
ERROR: It allows tracee to detect host environment features.
ERROR: 
ERROR: Run docker with :
ERROR:     -v /etc/os-release:/etc/os-release-host:ro
ERROR: 
ERROR: Then you may set LIBBPFGO_OSRELEASE_FILE:
ERROR:     -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host
ERROR:
``` 

<img width="1068" alt="Screenshot 2023-10-27 at 12 52 16" src="https://github.com/aquasecurity/tracee/assets/33576047/8aeaacbb-c42b-4a0a-a204-a945e30c5594">

Using `aarch64-dev` image:
```console
❯ docker run \
  --name tracee --rm -it \
  --pid=host --cgroupns=host --privileged \
  -v /etc/os-release:/etc/os-release-host:ro \
  aquasec/tracee:aarch64-dev
INFO: starting tracee...
{"level":"fatal","ts":1698407630.603109,"msg":"Tracee runner failed","error":"cmd.Runner.Run: error initializing Tracee: ebpf.(*Tracee).Init: ebpf.(*Tracee).NewKernelSymbols: invalid ksymbol table (capabilities issue ?)"}
```

Through vagrant and parallels, the same command with the latest image works